django_nitro_mailer/tasks.py

```diff
@@ -21,8 +21,12 @@ def send_emails(queryset: Optional[models.QuerySet] = None) -> None:
                 if email_message:
                     connection.send_messages([email_message])
 
-                    EmailLog.objects.create(email_data=email_obj.email_data, result=EmailLog.Results.SUCCESS)
+                    email_log = EmailLog.objects.create(
+                        email_data=email_obj.email_data, result=EmailLog.Results.SUCCESS
+                    )
+                    logger.info(str(email_log))
```

Comment:
Myślę, że możemy tutaj zrobić troszkę ładniejszą wiadomość:
```suggestion
                    logger.info("Email sent successfully", extra={"recipients": email_obj.recipients, "created_at": timezone.now()})
```